common/djangoapps/microsite_configuration/admin.py

```diff
@@ -15,4 +15,24 @@ class Meta(object):  # pylint: disable=missing-docstring
         model = Microsite
 
 
+class MicrositeHistoryAdmin(admin.ModelAdmin):
```

Comment:
Also, I'm not sure about if we need to keep an archive table or not. Right now, my sense is 'yes, we should'